bstackdemo.js

```diff
@@ -1,30 +1,33 @@
 require("chromedriver");
+require("geckodriver");
 
 let {By, Builder, Key, until, Capability} = require("selenium-webdriver");
-let {browserstack_user, browserstack_key} = require("../store")
-
-// var capabilities = {
-//     "os" : "Windows",
-//     "os_version" : "10",
-//     "browserName" : "Chrome",
-//     "browser_version" : "latest",
-//     "project" : "Bstack Demo test",
-//     "name" : "Bstack Demo test",
-//     "browserstack.local" : "false",
-//     "browserstack.networkLogs" : "true",
-//     "browserstack.selenium_version" : "3.14.0",
-//     "browserstack.user" : browserstack_user,
-//     "browserstack.key" : browserstack_key
-// }
-
-const capabilities = [
+const prompt = require("prompt-sync")({ sigint: true });
+
+let browserstack_user = prompt("Enter username: ");
```

Comment:
instead of taking input from users, we ask them to set username and access_key as environment variables and read from environment variables. Please update this to read from Environment variables.